test/hydration.js

```diff
@@ -45,6 +45,16 @@ describe('hydration', () => {
       result.should.eql('hydrate|||{"type":"method","method":"some.path.to","bundle":{}}|||hydrate');
     });
 
+    it('should not accept payload size bigger than 4096 bytes.', () => {
+      const inputData = { key: 'a'.repeat(4097) };
+      const payloadSize = JSON.stringify(inputData).length;
+      try {
+        dehydrate(funcToFind, inputData);
```

Comment:
I'd add a `should(false).eql(true)` to make sure this test fails if `dehydrate` allows a payload with `4097`.

Reply:
I've always wondered the purpose of `'1'.should.eql('2'); // shouldn't pass`. Would I be correct if I say `createDehydrator` isn't stable?

Reply:
It should be stable. That kind of assertion just makes sure that if someone changes the behavior of the tested method, then the test starts failing, otherwise, if the `try` doesn't throw an exception, the test will pass, not doing its job. Does that make sense?

Reply:
Makes perfect sense. Thanks for the explanation.